fe/src/stores/useSkillStore.ts

```diff
@@ -104,9 +104,12 @@ export const useSkillStore = defineStore('skills', {
             (skill) => skill.id === deletedSkillId
           )
           this.skills.splice(index, 1)
+          callback(true)
+        })
+        .catch(() => {
+          console.log('Failed to delete')
```

Comment:
The `console.log()` police is here! 🚨
Put down the log statement and come out with your hands above your head!